src/zfstools/connection.py

```diff
@@ -34,9 +34,11 @@ class ZFSConnection:
     _poolset = None
     _dirty = True
     _trust = False
-    def __init__(self,host="localhost", trust=False, sshcipher=None):
+    _properties = None
+    def __init__(self,host="localhost", trust=False, sshcipher=None, properties=[]):
```

Comment:
properties=[] is a textbook Python bug.  Needs to be None and then overridden with empty list if properties evaluates to false.